sound/soc/sof/intel/hda-ipc.c

```diff
@@ -113,6 +127,7 @@ irqreturn_t hda_dsp_ipc_irq_thread(int irq, void *context)
 	struct snd_sof_dev *sdev = (struct snd_sof_dev *)context;
 	u32 hipci, hipcie, hipct, hipcte, msg = 0, msg_ext = 0;
 	irqreturn_t ret = IRQ_NONE;
+	int reply = 1;
```

Comment:
snd_soc_ipc_reply returns 0 or -EINVAL, the 1 isn't incorrect but misleading. Change to -EINVAL for consistency?

Reply:
yes, it is right

Reply:
i will change it to -EINVAL